src/components/Messages/CreateMessageModal/MessagesModalListItem.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  ListItemButton,
+  ListItemAvatar,
+  Avatar,
+  ListItemText,
+  Typography,
+  Box,
+} from "@mui/material";
+import { OtherUser } from "./MessagesModalList";
+import { useNavigate } from "react-router-dom";
+import { useAppSelector } from "../../../state/hooks";
+
+const styles = {
+  avatar: { margin: "auto" },
+  displayName: {
+    fontWeight: "bold",
+  },
+  primaryTextContainer: {
+    gap: 0.5,
+    width: "30%",
+    height: "100%",
+  },
+  stack: {
+    display: "flex",
+    flexDirection: "row",
+    width: "100%",
+  },
+};
+
+type MessageModalListItemProps = {
+  otherUser: OtherUser;
+  onClose: () => void;
+};
+
+const MessageModalListItem = ({
+  otherUser,
+  onClose,
+}: MessageModalListItemProps) => {
+  const user = useAppSelector((state) => state.user);
+  const navigate = useNavigate();
+  const routeChange = () => {
+    onClose();
+    const path = `/messages/${user.userId}/${otherUser.otherUserId}`;
+    navigate(path);
+  };
+
+  return (
+    <ListItemButton onClick={() => routeChange()}>
+      <Box sx={styles.stack}>
```

Comment:
Nitpick, this isn't a stack anymore, so can we change the class name?

Reply:
changed it to `container`